Lib/importlib/_bootstrap.py

```diff
@@ -1306,8 +1306,15 @@ def _sanity_check(name, package, level):
 _ERR_MSG_PREFIX = 'No module named '
 
 def _find_and_load_unlocked(name, import_):
+    sys.audit(
+        "import",
+        name,
+        None,
```

Comment:
Either use the *path* variable, or remove it. I like the variable, so maybe just use it :-)

Reply:
The variable was already there and is used lower in the function, I don't really have an opinion on using it here or not. I'll add it back!